modules/oe_theme_content_entity_contact/config/overrides/core.entity_view_display.oe_contact.oe_general.oe_details.yml

```diff
@@ -0,0 +1,75 @@
+langcode: en
```

Comment:
As `oe_content_entity_contact` has been already released, we need to create a post update hook to create this configuration entity, in the same fashion as described here: https://github.com/openeuropa/oe_content/pull/228#discussion_r469183835